common/tools/dev-tool/src/commands/customization/apply-v2.ts

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import { resolveProject } from "../../util/resolveProject";
+import { createPrinter } from "../../util/printer";
+import { leafCommand } from "../../framework/command";
+import { makeCommandInfo } from "../../framework/command";
+
+import { SpawnOptions, spawn } from "node:child_process";
+import path from "node:path";
+import fs from "node:fs/promises";
+import os from "node:os";
+
+async function run(command: string[] | string, options: SpawnOptions = {}) {
```

Comment:
This could be useful as an util to use in other commands in the future.

Reply:
I extracted it into a util. There are a few places where we can probably shift to using that util already (test proxy stuff comes to mind), but I will tackle that in a separate PR.